### ISSUE TITLE
How to disable mouse wheel on slidebox

### ISSUE THREAD

**Issue by mukhtar-ali** [jtsage/jtsage-datebox#467]
My issue is when trying to scroll down on the page and the mouse happens to pass through the slidebox it's spinning and the scroll event happens to it instead of the page

I want to disable it in both month and days

Thanks in advance

**Comment by jtsage** [comment]
Sorry for the delay - from your report, I'm guessing you are using version 4 of datebox.  

Unfortunately, I'm not rolling new features for that version any longer, and have put my time into version 5.

Should you choose to upgrade, there is a new feature as of today called "wheelDisable".

If not, make a copy of the script locally (if you haven't aleady), and search for this bit of code:

```js
if ( typeof $.event.special.mousewheel !== "undefined" ) {
	w.wheelExists = true;
}
```

If you change that "true" to a "false", the mousewheel events will not bind.

**Comment by mukhtar-ali** [comment]
@jtsage Thank you for your kind replay, most appreciated it 👍 
I'll try it for sure and thanks for the awesome plugin.

**Comment by jtsage** [comment]
Re Version 5.1.1, SHA: df7153a30ca96dfada0b8ad300e879cf7b63a9b6

**Comment by jtsage** [comment]
Fix pushed to release, SHA: df7153a30ca96dfada0b8ad300e879cf7b63a9b6 (5.1.1)

(already superseded by 5.1.2, but adding here for historical purposes.)

Closing issue, if this comes back, or I missed something, feel free to re-open.
